[WPE] Rename WPEView as WPEWebView

carlosgcampos opened the pull request:
#### ea7d2765db2ffcddd0f20e2c51ca1f3e63ecacf1
<pre>
[WPE] Rename WPEView as WPEWebView
<a href="https://bugs.webkit.org/show_bug.cgi?id=265191">https://bugs.webkit.org/show_bug.cgi?id=265191</a>

Reviewed by Don Olmstead.

WPEView will conflict with new WPE platform API.

* Source/WebKit/SourcesWPE.txt:
* Source/WebKit/UIProcess/API/C/wpe/WKView.cpp:
* Source/WebKit/UIProcess/API/glib/WebKitUserContentManager.cpp:
* Source/WebKit/UIProcess/API/glib/WebKitWebView.cpp:
* Source/WebKit/UIProcess/API/wpe/PageClientImpl.cpp:
* Source/WebKit/UIProcess/API/wpe/WPEWebView.cpp: Renamed from Source/WebKit/UIProcess/API/wpe/WPEView.cpp.
(WKWPE::View::View):
(WKWPE::m_backend):
(WKWPE::View::~View):
(WKWPE::View::setClient):
(WKWPE::View::frameDisplayed):
(WKWPE::View::willStartLoad):
(WKWPE::View::didChangePageID):
(WKWPE::View::didReceiveUserMessage):
(WKWPE::View::webResourceLoadManager):
(WKWPE::View::setInputMethodContext):
(WKWPE::View::inputMethodContext const):
(WKWPE::View::setInputMethodState):
(WKWPE::View::selectionDidChange):
(WKWPE::View::setSize):
(WKWPE::View::setViewState):
(WKWPE::View::handleKeyboardEvent):
(WKWPE::View::synthesizeCompositionKeyPress):
(WKWPE::View::close):
(WKWPE::View::setFullScreen):
(WKWPE::View::accessible const):
(WKWPE::View::platformWebPageProxyForGamepadInput):
* Source/WebKit/UIProcess/API/wpe/WPEWebView.h: Renamed from Source/WebKit/UIProcess/API/wpe/WPEView.h.
(WKWPE::View::create):
(WKWPE::View::page):
(WKWPE::View::client const):
(WKWPE::View::backend):
(WKWPE::View::size const):
(WKWPE::View::viewState const):
(WKWPE::View::isFullScreen):
(WKWPE::View::touchGestureController const):
* Source/WebKit/UIProcess/API/wpe/WebKitPopupMenu.cpp:
* Source/WebKit/UIProcess/Gamepad/libwpe/UIGamepadProviderLibWPE.cpp:

Canonical link: <a href="https://commits.webkit.org/271034@main">https://commits.webkit.org/271034@main</a>
</pre>
<!--EWS-Status-Bubble-Start-->
https://github.com/WebKit/WebKit/commit/77f8c38755c45a1259cbb43b7a3cebab10d079af

| Misc | iOS, tvOS & watchOS  | macOS  | Linux |  Windows |
| ----- | ---------------------- | ------- |  ----- |  --------- |
| [✅ 🧪 style](https://ews-build.webkit.org/#/builders/38/builds/27091 "Passed style check") | [✅ 🛠 ios](https://ews-build.webkit.org/#/builders/48/builds/5707 "Built successfully") | [✅ 🛠 mac](https://ews-build.webkit.org/#/builders/14/builds/28327 "Built successfully") | [✅ 🛠 wpe](https://ews-build.webkit.org/#/builders/5/builds/29299 "Built successfully") | [✅ 🛠 wincairo](https://ews-build.webkit.org/#/builders/32/builds/24772 "Built successfully") 
| | [✅ 🛠 ios-sim](https://ews-build.webkit.org/#/builders/49/builds/7581 "Built successfully") | [✅ 🛠 mac-AS-debug](https://ews-build.webkit.org/#/builders/51/builds/3102 "Built successfully") | [✅ 🧪 wpe-wk2](https://ews-build.webkit.org/#/builders/34/builds/24631 "Passed tests") | 
| [✅ 🧪 webkitperl](https://ews-build.webkit.org/#/builders/11/builds/27353 "Passed tests") | [✅ 🧪 ios-wk2](https://ews-build.webkit.org/#/builders/47/builds/4526 "Passed tests") | [✅ 🧪 api-mac](https://ews-build.webkit.org/#/builders/18/builds/23253 "Passed tests") | [✅ 🧪 api-wpe](https://ews-build.webkit.org/#/builders/41/builds/3956 "Passed tests") | 
| | [✅ 🧪 ios-wk2-wpt](https://ews-build.webkit.org/#/builders/42/builds/4047 "Passed tests") | | [✅ 🛠 gtk](https://ews-build.webkit.org/#/builders/2/builds/29935 "Built successfully") | 
| | [✅ 🧪 api-ios](https://ews-build.webkit.org/#/builders/13/builds/24729 "Passed tests") | [✅ 🧪 mac-wk2](https://ews-build.webkit.org/#/builders/36/builds/24661 "Passed tests") | [✅ 🧪 gtk-wk2](https://ews-build.webkit.org/#/builders/1/builds/30248 "Passed tests") | 
| | [✅ 🛠 tv](https://ews-build.webkit.org/#/builders/44/builds/4059 "Built successfully") | [✅ 🧪 mac-AS-debug-wk2](https://ews-build.webkit.org/#/builders/50/builds/2250 "Passed tests") | [✅ 🧪 api-gtk](https://ews-build.webkit.org/#/builders/21/builds/28165 "Passed tests") | 
| | [✅ 🛠 tv-sim](https://ews-build.webkit.org/#/builders/46/builds/5523 "Built successfully") | | | 
| | [✅ 🛠 watch](https://ews-build.webkit.org/#/builders/43/builds/4524 "Built successfully") | | | 
| [✅ 🛠 🧪 unsafe-merge](https://ews-build.webkit.org/#/builders/22/builds/3510 "Built successfully and passed tests") | [✅ 🛠 watch-sim](https://ews-build.webkit.org/#/builders/45/builds/4436 "Built successfully") | | | 
<!--EWS-Status-Bubble-End-->

mcatanzaro reviewed (commented):
How about View.h, so it matches the name of the class?

It's not a great name, but it's really confusing when the filename doesn't match the name of the class.

carlosgcampos committed:
[WPE] Rename WPEView as WPEWebView
https://bugs.webkit.org/show_bug.cgi?id=265191

Reviewed by Don Olmstead.

WPEView will conflict with new WPE platform API.

* Source/WebKit/SourcesWPE.txt:
* Source/WebKit/UIProcess/API/C/wpe/WKView.cpp:
* Source/WebKit/UIProcess/API/glib/WebKitUserContentManager.cpp:
* Source/WebKit/UIProcess/API/glib/WebKitWebView.cpp:
* Source/WebKit/UIProcess/API/wpe/PageClientImpl.cpp:
* Source/WebKit/UIProcess/API/wpe/WPEWebView.cpp: Renamed from Source/WebKit/UIProcess/API/wpe/WPEView.cpp.
(WKWPE::View::View):
(WKWPE::m_backend):
(WKWPE::View::~View):
(WKWPE::View::setClient):
(WKWPE::View::frameDisplayed):
(WKWPE::View::willStartLoad):
(WKWPE::View::didChangePageID):
(WKWPE::View::didReceiveUserMessage):
(WKWPE::View::webResourceLoadManager):
(WKWPE::View::setInputMethodContext):
(WKWPE::View::inputMethodContext const):
(WKWPE::View::setInputMethodState):
(WKWPE::View::selectionDidChange):
(WKWPE::View::setSize):
(WKWPE::View::setViewState):
(WKWPE::View::handleKeyboardEvent):
(WKWPE::View::synthesizeCompositionKeyPress):
(WKWPE::View::close):
(WKWPE::View::setFullScreen):
(WKWPE::View::accessible const):
(WKWPE::View::platformWebPageProxyForGamepadInput):
* Source/WebKit/UIProcess/API/wpe/WPEWebView.h: Renamed from Source/WebKit/UIProcess/API/wpe/WPEView.h.
(WKWPE::View::create):
(WKWPE::View::page):
(WKWPE::View::client const):
(WKWPE::View::backend):
(WKWPE::View::size const):
(WKWPE::View::viewState const):
(WKWPE::View::isFullScreen):
(WKWPE::View::touchGestureController const):
* Source/WebKit/UIProcess/API/wpe/WebKitPopupMenu.cpp:
* Source/WebKit/UIProcess/Gamepad/libwpe/UIGamepadProviderLibWPE.cpp:

Canonical link: https://commits.webkit.org/271034@main
